File: resources/python/throw-try-catch-finally.py
try:
    with open('thisfiledoesnotexist.txt', 'r') as f:
        read_data = f.read()
except FileNotFoundError as e:
    print(e)
except Exception as e:
    print(e)
else:
    raise NameError("hi there")
finally:
    print("the end")

File: resources/kotlin/throw-try-catch-finally.kt
import java.io.File
import java.io.IOException

fun main() {
    try {
        File("thisfiledoesnotexist.txt").readText(Charsets.UTF_8)
        throw RuntimeException("lets throw a RuntimeException")
    } catch (exception: IOException) {
        println(exception.message)
    } catch (exception: Exception) {
        println(exception.message)
    } finally {
        println("finally")
    }
}
